modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -487,15 +496,39 @@ private boolean _equals(
 					}
 				}
 
-				if (childrenGraphQLType1.size() ==
-						childrenGraphQLType2.size()) {
+				if (childrenGraphQLFieldDefinitions1.size() ==
+						childrenGraphQLFieldDefinitions2.size()) {
 
 					return true;
 				}
 
 				return false;
 			}
 
+			private List<GraphQLFieldDefinition> _getGraphQLFieldDefinitions(
+				GraphQLType graphQLType1) {
+
+				List<GraphQLType> children = graphQLType1.getChildren();
+
+				Stream<GraphQLType> stream = children.stream();
+
+				return stream.filter(
+					GraphQLFieldDefinition.class::isInstance
+				).map(
+					GraphQLFieldDefinition.class::cast
+				).collect(
+					Collectors.toList()
+				);
+			}
+
+			private String _getGraphQLFieldDefinitionTypeName(
+				GraphQLFieldDefinition childGraphQLFieldDefinition1) {
+
+				GraphQLOutputType type = childGraphQLFieldDefinition1.getType();
+
+				return type.toString();
```

Comment:
I've used toString method here, because the getName method returns null with GraphQL wrapper types (just the case of the items field inside pages)